addon/components/high-charts.js

```diff
@@ -39,40 +39,53 @@ export default Component.extend({
   didReceiveAttrs() {
     this._super(...arguments);
 
-    if (!(get(this, 'content') && get(this, 'chart'))) {
+    const { content, chart, mode } = this.getProperties('content', 'chart', 'mode');
+    if (!content || !chart) {
       return;
     }
 
-    let chart = get(this, 'chart');
     let noData = chart.get('noData');
```

Comment:
I see. Thanks for explaining. I'll work on adding a test for this "noData" use case.